cli/cmd/version.go

```diff
@@ -61,4 +68,6 @@ func printOdigosClusterVersion(cmd *cobra.Command) {
 
 func init() {
 	rootCmd.AddCommand(versionCmd)
+
+	versionCmd.PersistentFlags().Bool("short", false, "prints only the CLI version")
```

Comment:
Why use PersistentFlags? I see that in all the other commands we are using Flags